groupsforsections.py

```diff
@@ -149,6 +149,33 @@ def create_group(groups_for_section, group_category_id, group_name, course_id):
     return str(group['id'])
 
 
+def add_users_to_group(groups_for_section, group_id, user):
+    """
+    adding user to a group, parses Json, do exception handling
+    :param groups_for_section:
+    :param group_id:
+    :param user:
+    :type groups_for_section GroupsForSections
+    :type group_id str
+    :type user str
+    :return: membership_id
+    :rtype: int, None
+    """
+    logging.debug(add_users_to_group.__name__ + '() called')
+    try:
+        response = groups_for_section.add_users_to_group(group_id, user)
+
+    except requests.exceptions.RequestException as e:
+        logging.exception('adding USER to group has erroneous response ' + e.message)
+        return None
+
+    if not handle_request_if_failed(response):
+        return
```

Comment:
Dave's question makes sense.  This looks odd to developers without much Python experience.

Unlike Java or C, in which a function's `return` statement without arguments doesn't return anything (`void`), Python will return a `None` object.  That's Python's equivalent to `null`, somewhat.  Python doesn't have a `null` and it's `None` is actually an instantiation of a class.  (Interestingly, Python may give different results for `x == None` and `x is None`.)

While experienced Python developers will not be surprised by `return` without arguments, it doesn't hurt to be explicit and use `return None`.  You should consider whether it's better to raise an exception rather than return `None`.